src/App.js

```diff
@@ -0,0 +1,88 @@
+import './App.css';
+import Content from './Content.js';
+import Header from './Header.js'
+import Footer from './Footer.js'
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import { Layout } from 'antd';
+
+const App = () => {
```

Comment:
This Component nearly contains the logic of all of the application which defy the concept of Single Responsibility Principle that components should abide by. You will find more info in later comments.